studio/pages/project/[ref]/logs-explorer/saved.tsx

```diff
@@ -1,37 +1,27 @@
 import React, { useEffect } from 'react'
-import { NextPage } from 'next'
 import Link from 'next/link'
 import { observer } from 'mobx-react-lite'
 import { IconSave, Loading } from '@supabase/ui'
-import { useStore, withAuth } from 'hooks'
+import { useStore } from 'hooks'
 import { LogsSavedQueriesItem } from 'components/interfaces/Settings/Logs'
 import LogsExplorerLayout from 'components/layouts/LogsExplorerLayout/LogsExplorerLayout'
 
 import Table from 'components/to-be-cleaned/Table'
 import { useRouter } from 'next/router'
+import { NextPageWithLayout } from 'types'
 
-export const LogsSavedPage: NextPage = () => {
-  const { content, ui } = useStore()
+export const LogsSavedPage: NextPageWithLayout = () => {
+  const { content } = useStore()
   const router = useRouter()
   const { ref } = router.query
 
-  useEffect(() => {
-    if (!content.isLoading) {
-      content.load()
-    }
-  }, [ui.selectedProject])
```

Comment:
This same `content.load()` happens in the layout, so it was causing an infinite loop. Not sure why it wasn't breaking before, but now it's gone.